tests/test_endpoints.py

```diff
@@ -54,6 +55,26 @@ def assertEndpointReached(self, func, params, method, endpoint, mock_request, ti
             timeout=timeout,
         )
 
+    @patch('myob.managers.requests.request')
+    def assertEmptyDeleteResponseHandled(self, func, params, endpoint, mock_request, timeout=None):
+        mock_request.return_value.status_code = 200
+
+        def response_json():
+            return Response().json()
+
+        mock_request.return_value.content = b''
+        mock_request.return_value.json = response_json
+        func(**params)
+        full_endpoint = 'https://api.myob.com/accountright' + endpoint
+        mock_request.assert_called_once_with(
+            'DELETE',
+            full_endpoint,
+            headers=self.expected_request_headers,
+            params={},
+            **({}),
+            timeout=timeout,
+        )
```

Comment:
This isn't quite right.
Our intention is to ensure that if calling `response.json()` raises a JSONDecodeError, it gets handled gracefully and we return empty dict instead.

It seems you're just testing what the request function gets called with.

Reply:
Well, not quite. With the old code the test blows up. But yes, I'll add a check for the empty dict return.